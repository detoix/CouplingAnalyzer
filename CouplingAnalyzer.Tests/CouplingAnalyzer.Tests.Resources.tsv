FromProject	FromFile	FromType	ToProject	ToFile	ToType
ChildProject	CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.VeryComplexChild	ChildProject	CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.ComplexChild
ParentProject	CouplingAnalyzer.Tests.Resources/ParentProject/OtherClasses.cs	ParentProject.Others.Test	ChildProject	CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.ComplexChild
ParentProject	CouplingAnalyzer.Tests.Resources/ParentProject/Parent.cs	ParentProject.Parent	ChildProject	CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.ComplexChild
ParentProject	CouplingAnalyzer.Tests.Resources/ParentProject/Parent.cs	ParentProject.Parent	ChildProject	CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.VeryComplexChild
ParentProject	CouplingAnalyzer.Tests.Resources/ParentProject/Parent.cs	ParentProject.IParent	ChildProject	CouplingAnalyzer.Tests.Resources/ChildProject/Extensions.cs	ChildProject.Extensions.OtherChild
