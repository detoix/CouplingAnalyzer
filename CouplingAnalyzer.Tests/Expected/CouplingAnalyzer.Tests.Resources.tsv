FromProject	FromFile	FromType	ToProject	ToFile	ToType
ChildProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.VeryComplexChild	ChildProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.ComplexChild
ParentProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ParentProject/OtherClasses.cs	ParentProject.Others.Test	ChildProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.ComplexChild
ParentProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ParentProject/Parent.cs	ParentProject.Parent	ChildProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.ComplexChild
ParentProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ParentProject/Parent.cs	ParentProject.Parent	ChildProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ChildProject/Child.cs	ChildProject.VeryComplexChild
ParentProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ParentProject/Parent.cs	ParentProject.IParent	ChildProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ChildProject/Extensions.cs	ChildProject.Extensions.OtherChild
ParentProject	/home/detoix/Software/CouplingAnalyzer/CouplingAnalyzer.Tests.Resources/ParentProject/Parent.cs	ParentProject.ParentData	Unknown	Unknown	ChildProject.Extensions.Extensions
